tests/helixpublishwitharcade.proj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk">
+  <PropertyGroup>
+    <HelixSource>pr/coreclr/master</HelixSource>
```

Comment:
When we do implement this, do we want these to be properties of the pipeline? Not sure, but something like /p:pr/coreclr/$(Build.SourceBranchName)? This way we can extend the workflow for more testing scenarios.